src/Fantomas.Core/CodeFormatterTypes.fs

```diff
@@ -1,6 +1,9 @@
 ﻿namespace Fantomas.Core
 
 open FSharp.Compiler.Text
+open System
+
+type ProfileInfos = { LineCount: int; TimeTaken: TimeSpan }
```

Comment:
No changes should be more to Fantomas.Core for this feature.
It was always a command line-only thing and I would like to keep it that way for now.

Reply:
Oops, yeah, changed that.